components/graphs/Graph.tsx

```diff
@@ -132,18 +179,28 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
               ) => weekdayFormatter.format(new Date(point))}
             />
             <Line
+              data={clearSkyEstimateTrimmed}
+              type="monotone"
+              dataKey="clearsky_generation_kw"
+              stroke="#48B0DF"
+              dot={false}
+              activeDot={{ r: 8 }}
+              onAnimationEnd={() => setTimeEnabled(true)}
+            />
+            <Line
+              data={forecastDataTrimmed}
               type="monotone"
               dataKey="expected_generation_kw"
               stroke="#FFD053"
               dot={false}
               activeDot={{ r: 8 }}
-              onAnimationEnd={() => setTimeEnabled(true)}
```

Comment:
Does the clearsky estimate always animate slower (or equal time to) the forecast? If so, keeping this deleted should be good for now.